### PR TITLE
Simplified requirements.

### PR DESCRIPTION
**Pull request by frisso**
Additional notes (to be discussed in person):
- not clear why we should select in-band vs outofband (is there any advantage?)
- it would be nice to create a picture showing the workflow
- not clear the meaning of the command "--context=consumer" in the peering

### CODE REVIEW

**Comment by frisso** [comment]
@giorio94 Feel free to add comments. If it's ok, I would merge this and leave the open points for a future PR.

File: docs/installation/requirements.md [hunk 1]
```diff
@@ -4,25 +4,27 @@ This page presents an overview of the main requirements, both in terms of **reso
...
+
+* [**Out-of-band control plane peering**](FeaturesPeeringOutOfBandControlPlane): it requires **three separated traffic flows** (hence, three exposed endpoints).
+* [**in-band control plane peering**](FeaturesPeeringInBandControlPlane): it requires a **single endpoint**, as all control plane traffic is tunneled inside the cross-cluster VPN.
+
+More details available in the [peering section](/features/peering).
 
 ```{admonition} Note
-The two peering approaches are **non-mutually exclusive**.
-In other words, a single cluster can leverage different approaches toward different remote clusters, in case all connectivity requirements are fullfilled.
+The two peering approaches are **non-mutually exclusive**: a cluster can leverage different approaches toward different remote clusters, provided that the connectivity requirements are verified.
```
Comment (giorio94) [inline docs/installation/requirements.md:27]: To me, *verified* conveys a different meaning compared to the original *fulfilled*. Is this intentional?

**Review by giorio94 — APPROVED** [review]

**Comment by giorio94** [comment]
/hold

Holding, as a remainder to squash the commits before merging.

**Pull request merged**